app.pianocheetah.pianocheetah.json

```diff
@@ -0,0 +1,31 @@
+{
+   "app-id":  "app.pianocheetah.pianocheetah",
+   "command": "pianocheetah",
+   "sdk":     "org.kde.Sdk",
+   "runtime": "org.kde.Platform",
+   "runtime-version": "6.6",
+   "modules": [
+      {
+         "name": "cp_to_app",
+         "buildsystem": "simple",
+         "build-commands": [
+            "cp -pr share /app",
+            "cp -pr bin   /app"
+         ],
+         "sources": [
+            {
+               "type": "dir",
+               "path": "_build/files"
+            }
+         ]
+      }
+   ],
+   "finish-args": [
+      "--filesystem=host",
```

Comment:
why?

Reply:
I really don't like --filesystem=home because when I display paths that the file chooser dialog gives me, it's:

/run/user/1000/doc/53548332/pianocheetah

that's pretty annoying.  I guess I can hide that stuff from the user, but my code is not setup to hide paths from the user.  So I really prefer --filesystem=system 
My program uses a top level directory for song files, sample files, midi clips, text files, etc
   that's pickable on first run.
And I typically put it on a different disk than my home dir.
And I really can't stand the names of those paths.

Is this permissible?